### PR TITLE
Release: 0.1.8

### PR DESCRIPTION
**Pull request by maxhutch**
The logic of the parsers hasn't changed; this is adding surrounding functionality: 
 - Building out convert method to handle in-line report generation
 - Adding Dockerfile and description.yaml for Citrine ingest pipeline
 - Pinning pypif version (since the new version changes the API)
 - Typo in "pseudopotentials"
 - Move drivers out of `__init__.py`

### CODE REVIEW

File: dfttopif/drivers.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        # Add it to the output
+        chem.properties.append(prop)
+
+    # Check to see if we should add the quality report
+    if quality_report and isinstance(parser, VaspParser) :
+        _add_quality_report(directory, chem)
+
+    return chem
+
+def convert(files=None, **kwargs):
```
Comment (maxhutch) [inline dfttopif/drivers.py]: @sinistral this method actually changed

Comment (sinistral) [inline dfttopif/drivers.py]: Thanks for the pointer!

Comment (sinistral) [inline dfttopif/drivers.py]: Is "None" a legal default?  If this parameter actually isn't specified, the conditional on l198 will break (can't take the `len` of `None`).

File: dfttopif/drivers.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    :param files: a list of files, should only have one file here
+    :param kwargs: any additional keyword arguments
+    :return: the created pif
+    """
+    if (len(files) == 1):
+        return directory_to_pif(files[0], **kwargs)
+    else:
+        prefix = os.path.join(".", os.path.commonprefix(files))
+        print("Trying to use prefix {} from {}".format(prefix, os.getcwd()))
+        return directory_to_pif(prefix, **kwargs)
```
Comment (sinistral) [inline dfttopif/drivers.py]: What will happen if `files` is empty?

File: description.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# This file describes the converter
+name: vasp                       # Any valid python key (should be unique)
```
Comment (sinistral) [inline description.yaml]: You should add a namespace to identify this as a Citrine converter, too.

**Pull request merged**